src/Iterable/windowed.php

```diff
@@ -24,9 +24,9 @@
  *
  * Both {@see $size} and {@see $step} must be positive and can be greater than the number of elements in this collection.
  *
- * @param array $iterable Collection to apply the windows
- * @param int   $size     The number of elements to take in each window
- * @param int   $step     The number of elements to move the window forward by on each step, by default 1
+ * @param mixed[] $iterable Collection to apply the windows
```

Comment:
why not `iterable`?

Reply:
Code standard expects to be specific about the type of collections.